app/src/main/java/com/example/yard/Post.java

```diff
@@ -0,0 +1,14 @@
+package com.example.yard;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+public class Post extends AppCompatActivity {
```

Comment:
Naming is important for code quality. If it's an activity, it should be named as `PostActivity`